include/event_manager.h

```diff
@@ -30,6 +30,13 @@
 extern "C" {
 #endif
 
+/** @brief Pointer to the event handler function.
+ *
+ * @param eh  Pointer to the event header of the event that is processed by Event Manager.
+ * @retval    0 If the operation was successful. Error values can be added by user.
```

Comment:
`true if event was consumed and should not be propagated to other listeners, false otherwise.`